package.json

```diff
@@ -60,19 +61,23 @@
   "scripts": {
     "compile": "hardhat compile",
     "run:node": "hardhat node",
-    "test": "cp .openzeppelin/avalanche.json .openzeppelin/unknown-31337.json && hardhat test",
-    "test:localhost": "cp .openzeppelin/avalanche.json .openzeppelin/unknown-31337.json && hardhat test --config ./hardhat.config.ts --network localhost",
+    "pretest": "cp .openzeppelin/avalanche.json .openzeppelin/unknown-31337.json",
+    "test": "hardhat test",
```

Comment:
I don't think so because it's how npm is designed to work. https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

Because the cp stuff needs to be executed 'before' test and I think specifying it as 'pretest' delivers this well.